src/manifold/include/manifold.h

```diff
@@ -47,16 +47,20 @@ class CsgLeafNode;
 struct MeshGL {
   /// Number of property vertices
   uint32_t NumVert() const {
+#if MANIFOLD_EXCEPTIONS 
     if (vertProperties.size() / numProp >=
         std::numeric_limits<unsigned int>::max())
       throw std::out_of_range("mesh too large");
+#endif
     return vertProperties.size() / numProp;
   };
   /// Number of triangles
   uint32_t NumTri() const {
+#if MANIFOLD_EXCEPTIONS 
     if (vertProperties.size() / numProp >=
         std::numeric_limits<unsigned int>::max())
       throw std::out_of_range("mesh too large");
+#endif
```

Comment:
I think I don't understand something about C++ exceptions. I thought any `throw` just automatically became `exit` when exceptions are disabled in the compiler. Or does disabling exceptions just turn uses of `throw` into compile errors?

Reply:
I vaguely recall that emscripten requires no `throw` statement to compile with exception disabled. Not really sure about that.

Reply:
Okay, but how about in normal C++ workflows? I still don't really understand #804. Is there something we did that made it impossible to build with exceptions disabled? @fire - can you help me understand https://github.com/godotengine/godot-proposals/issues/9711#issuecomment-2146063161? Our exceptions at this point are only a convenience for debug and such. Is there something that breaks when you compile with exceptions disabled?

Reply:
When we compile with exceptions enabled, for example web, and a few of the platforms break. We couldn't disable exceptions.

Reply:
Ah, so this is specific to web. In that case this PR may well address the issue. We could really use this kind of detail here or in #804. Which platforms break? What were the errors? Why couldn't you disable exceptions? Do you want to disable exceptions? 

I really want to help you out, but I do need at least *some* feedback.

Reply:
Here's what I can do. I'll push the branch with exceptions on and watch godot engine cid crash. Will post the branch.

Reply:
https://github.com/V-Sekai/godot/tree/vsk-csg-exceptions-4.3